src/main/java/frc/robot/ControllerBindings.java

```diff
@@ -8,14 +8,19 @@
 public class ControllerBindings {
 
     private LogitechDualAction driverPad;
+    private LogitechDualAction operatorPad;
 
     public static final LogitechControl DRIVE_LEFT_STICK = LogitechControl.LEFT_STICK;
     public static final LogitechControl DRIVE_RIGHT_STICK = LogitechControl.RIGHT_STICK;
 
-    ControllerBindings(int driverPort){
+    ControllerBindings(int driverPort, int operatorPort){
```

Comment:
Seems kinda funny to pass these in as constructor parameters when this class exists for the sole purpose of defining everything else as constants. Why not make these constants within this class too?